examples/webgl_loader_pdb.html

```diff
@@ -108,16 +109,16 @@
 
 			function init() {
 
-				camera = new THREE.PerspectiveCamera( 70, window.innerWidth / window.innerHeight, 1, 5000 );
-				camera.position.z = 1500;
-
 				scene = new THREE.Scene();
 
-				var light = new THREE.HemisphereLight( 0xffffff, 0x000000 );
-				light.position.set( 1, 1, 1 );
```

Comment:
The new `PointLight` is a child of the camera now. That way, when the `TraclballControls` rotates the camera, the molecules are always evenly lit. The point light provides a small specular highlight similar to what was there before. If you want to do something different, that is OK with me.